### PR TITLE
Gha

### PR DESCRIPTION
**Pull request by jburel**
* benchmak tests with tiff files are still failing despite having the latest version of zarr library installed (2.7.0)
* install dependencies via conda to avoid the downgrade of packages like ``s3fs``.

Edit: Tests are now passing for Tiff. Handling another type of Error 
i have tried to set the path to ``None`` which is supposed to be the default but this does not work.

### CODE REVIEW

**Comment by jburel** [comment]
https://github.com/jburel/ngff-latency-benchmark/actions/runs/732159644 is green with the last commit

**Comment by jburel** [comment]
https://github.com/jburel/ngff-latency-benchmark/actions/runs/732462261 green with 554e22e03b08c5076646f67d547f265ac3bd2f7d

File: benchmark/environment.yml [hunk 1]
```diff
@@ -16,9 +17,10 @@ dependencies:
   - zarr >= 2.7.0
   - go-mc
   - pip
+  - boto3
```
Comment (joshmoore) [inline benchmark/environment.yml:20]: These changes are all needed with your `ValueError` fix?

Comment (jburel) [reply benchmark/environment.yml:20]: I don't think so.
I will roll back.

**Comment by jburel** [comment]
https://github.com/jburel/ngff-latency-benchmark/actions/runs/732879227 is green

**Comment by jburel** [comment]
https://github.com/jburel/ngff-latency-benchmark/actions/runs/735408069 is green with the last commit i.e. special env for gha

**Comment by joshmoore** [comment]
`.env-gha` is a good idea. Thanks.

```
+ python plot_results.py
status,duration,chunk_distance,chunk_index,round,seconds,source,type
passed,0.00017480099995736964,494,"[1, 1, 1, 16, 15]",0,2.0265579223632812e-05,local,Overhead
passed,0.0010079070000301726,494,"[1, 1, 1, 16, 15]",0,0.0008678436279296875,http,Overhead
passed,0.008338460000004488,494,"[1, 1, 1, 16, 15]",0,0.00815129280090332,s3,Overhead
passed,0.00011680100010380556,553,"[1, 1, 1, 18, 10]",1,1.1682510375976562e-05,local,Overhead
passed,0.0008410059999732766,553,"[1, 1, 1, 18, 10]",1,0.0007195472717285156,http,Overhead
passed,0.00680754900008651,553,"[1, 1, 1, 18, 10]",1,0.006684064865112305,s3,Overhead
passed,0.00010520099999666854,159,"[1, 1, 1, 5, 32]",2,1.0967254638671875e-05,local,Overhead
passed,0.0007899059999090241,159,"[1, 1, 1, 5, 32]",2,0.0006706714630126953,http,Overhead
passed,0.006509546999950544,159,"[1, 1, 1, 5, 32]",2,0.006385326385498047,s3,Overhead
```
:+1:

I'm unsure why the commits are being tested on this PR, but merging with the hope that it activates the action.

**Comment by jburel** [comment]
This will now be active on the repo. The first time, we have to use the "Actions" on the fork of the authors.

**Pull request merged**